docs/faq.md

```diff
@@ -194,11 +190,24 @@ can use `oc adm release info` to get the `machine-os-content` image URL for a
 particular release, and then e.g. use `oc image extract` or `podman create` +
 `podman copy` to extract the RPMs.
 
-In 4.13 and later, extensions are shipped as a separate image. The image label is
-`rhel-coreos-8-extensions` and the RPMs are located in `/usr/share/rpm-ostree/extensions`.
+In 4.13 and later, extensions are shipped as a separate image. The image name is
+`rhel-coreos-extensions` and the RPMs are located in `/usr/share/rpm-ostree/extensions`.
 The container also works as an HTTP server serving repodata containing the extensions
 RPMs (port 9091).
 
+In 4.19 and later, it's possible to cheaply query the extensions list:
+
+```bash
+$ oc image extract "$(oc adm release info --image-for=rhel-coreos-extensions quay.io/openshift-release-dev/ocp-release:4.19.0-rc.3-x86_64)[-1]" --file usr/share/rpm-ostree/extensions.json
```

Comment:
What's the `[-1]` for here?

```suggestion
$ oc image extract "$(oc adm release info --image-for=rhel-coreos-extensions quay.io/openshift-release-dev/ocp-release:4.19.0-rc.3-x86_64)" --file usr/share/rpm-ostree/extensions.json
```

Reply:
https://github.com/okd-project/okd/releases/download/4.19.0-okd-scos.1/sha256sum.txt

Reply:
ahh. it means to extract the last layer in the image